src/components/Header/Header.js

```diff
@@ -2,19 +2,18 @@ import React from 'react';
 import HeaderNav from './HeaderNav/HeaderNav';
 import HeaderMessages from './HeaderMessages/HeaderMessages';
 import HeaderUserMenu from './HeaderUserMenu/HeaderUserMenu';
-import headerData from '../../headerData.json';
 import { FormattedMessage } from 'react-intl';
 
-const Header = () => {
+const Header = ({ userData: { user, nav, alert } }) => {
```

Comment:
its better than use props?

Reply:
In fact, we are using `props` here.

`props` object is being deconstructed. 

Given:

```js
const props = {
  userData: {
    user: {
        // . . . 
    }, 
    nav: {
        // . . . 
    },  
    alert: {
        // . . . 
    }, 
    another_property_1: : {
        // . . . 
    }, 
  },
  another_property_2: : {
        // . . . 
    }, 
};
```

This:

```js
const { userData: { user, nav, alert } } = props;
```

Is the same than:

```js
const user = props.userData.user;
const nav = props.userData.nav;
const alert = props.userData.alert;
```

So, it is shorter. But **the main advantage is** that it is clear what data we are using: we cannot use other data from _props_ without making it explicit. Does it make sense?

CC: @cbernat who was interested in deconstructing.

Reply:
Yes, i understand now